test/functionalTest/cases/1129_strings_in_q_uri_param/strings_in_q_uri_param.test

```diff
@@ -0,0 +1,428 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Strings in q URI Param
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create E1 with a String attribute S == '123'
+# 02. Create E2 with a Number attribute S == 123
+# 03a. Query with q=S:'123', see E1
+# 03b. Query with q=S=='123', see E1
+# 04a. Query with q=S:123, see E2
+# 04b. Query with q=S==123, see E2
+# 05a. Query with int-list: q=S:121,122,123 - see E2
+# 05b. Query with int-list: q=S:121,122,124 - see no hit
+# 06a. Query with string-list: q=S:'121','122','123' - see E1
+# 06b. Query with string-list: q=S:'121','122','124' - see no hit
+#
+# 07. Create subscription with q=S:121,122,123
```

Comment:
Steps 7 to 10 test that the filter works ok for the intial notification. However, it would be also interesting to check that they work when the attribute covered by the subscription filter is updated.

In order not making this .test too large, maybe the subscription test should be moved to a different .test.